ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -391,6 +391,8 @@ objects:
         cat > /tmp/tf/terraform.tfvars <<-EOF
           ipam = "139.178.89.254"
 
+          machine_cidr = "139.178.89.192/26"
```

Comment:
This particular CIDR corresponds to the vcsa.vmware.devcluster.openshift.com.
The ci environment vcsa-ci.vmware.devcluster.openshift.com uses: "139.178.73.0/26".
The qe environment vcsa-qe.vmware.devcluster.openshift.com uses:  "139.178.76.0/26"

The vcenter_server environment should correspond to those CIDR.

Reply:
Ah, thanks! Eventually this will be a part of vsphere secret anyway